en/identity-server/7.0.0/docs/deploy/security/security-guidelines/product-level-security-guidelines.md

```diff
@@ -227,7 +227,7 @@ Log forging can be prevented by appending a UUID to the log message.
 
 ## JVM parameters
 
-- The recommended JDK versions are JDK 1.8 and 11. For more information, see <a href="{{base_path}}/deploy/get-started/install/#prerequisites">Prerequisites</a>. 
+- The recommended JDK versions are JDK 11 and 17. For more information, see <a href="{{base_path}}/deploy/get-started/install/#prerequisites">Prerequisites</a>. 
```

Comment:
I think we should refer to the https://is.docs.wso2.com/en/next/deploy/environment-compatibility/#tested-operating-systems-and-jdks section without mentioning the specific versions here.